benchmark/compare.py

```
@@ -0,0 +1,204 @@
+    print(f"  Accuracy  : {m['accuracy']:.1%}  ({int(m['accuracy']*total)}/{total})")
+    print(f"  Precision : {m['precision']:.1%}")
+    print(f"  Recall    : {m['recall']:.1%}")
+    print(f"  F1        : {m['f1']:.3f}")
+    print(f"  FP        : {m['fp']} / {m['nomatch_n']}  ({m['fp']/m['nomatch_n']:.0%} of no-match)")
+    print(f"  FN        : {m['fn']} / {m['match_n']}  ({m['fn']/m['match_n']:.0%} of match)")
+    print(f"  Latency   : median={statistics.median(latencies):.2f}ms  "
+          f"p95={s[int(len(s)*.95)]:.2f}ms  max={s[-1]:.2f}ms")
+
+    issues = sorted(set(m["per_fn"]) | set(m["per_fp"]))
+    if issues:
+        print(f"\n  Per-intent (issues only):")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)